### ISSUE TITLE
 Anchor and left mouse click doesn't work as expected

### ISSUE THREAD

**Issue by kurotych** [DioxusLabs/dioxus#795]
**Problem**

When I click by left mouse button on tittle with `href="#philosophy"`  instead to scroll to `Philosophy` tittle, It trying to be opened in my Firefox browser.

But when I use the right mouse button (in dev build) and "Open link" menu it works as expected 

**Steps To Reproduce**

# Initial state
![image](https://user-images.githubusercontent.com/20345096/213943193-812b8cc5-a32c-4558-8d2b-3b6fdb3a04b2.png)
------------
![image](https://user-images.githubusercontent.com/20345096/213943203-ebccdaa0-5ca9-4e99-864a-0f45ee14b1c6.png)

# Result after the left mouse button click
![image](https://user-images.githubusercontent.com/20345096/213943228-4ab6fcd8-5025-4952-b910-8d2e5bda0a8c.png)

# Result after the right mouse button and "Open link"
![image](https://user-images.githubusercontent.com/20345096/213943409-05eaff63-8968-4efb-81b1-244b1c156f66.png)

P.S. All HTML content generated by `dangerous_inner_html` option 

**Expected behavior**

A left mouse button click will work as right mouse button with "Open link"
In other words: I expected to be scrolled to the needed Tittle with clicking the left mouse button

**Screenshots**

**Environment:**
 - Dioxus version: master
 - Rust version: 1.65
 - OS info: Linux Debian (Wayland)
 - App platform: Desktop
 
**Questionnaire**

- [x] I'm interested in fixing this myself but don't know where to start
- [ ] I would like to fix and I have a solution
- [ ] I don't have time to fix this right now, but maybe later

**Comment by kurotych** [comment]
UPD: looks like the problem has gone o_O 
Maybe because I've updated to the latest master branch version of the project.
